tests/CLI/CLI.cs

```diff
@@ -13,6 +13,11 @@ public CLITestsGenerator(GeneratorKind kind)
 
         public override void Setup(Driver driver)
         {
+            driver.Options.GenerateName = file =>
```

Comment:
I guess this is set because you changed it in such a way that it crashes if GenerateName is not set. But as indicated above, this is not correct. Remove this piece, it must work without it.